src/mixins/translatable.js

```diff
@@ -35,6 +36,20 @@ export default {
     listeners () {
       window.addEventListener('scroll', this.translate, false)
       window.addEventListener('resize', this.translate, false)
+
+      let isFirefox = window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1
```

Comment:
is this browser detection required?

Reply:
My solution isn't very optimized, i added browser check to don't use it in other browsers